lms/djangoapps/lti_provider/tests/test_views.py

```diff
@@ -206,3 +207,23 @@ def test_unauthenticated(self):
         self.setup_course()
         self.setup_user(admin=False, enroll=True, login=False)
         self.verify_response()
+
+    def test_hidden_bookmarks(self):
+        """
+        LTI view, which uses the "chromeless" view, should not support bookmarking
+        """
+        self.setup_course()
+        chapter = ItemFactory.create(parent=self.course, category='chapter')
+        self.vetical_block = ItemFactory.create(  # pylint: disable=attribute-defined-outside-init
+            parent=chapter,
+            category='vertical'
+        )
+        for i in xrange(2):
+            ItemFactory.create(  # pylint: disable=attribute-defined-outside-init
+                parent=self.vetical_block,
+                category='html',
+                data="<p>Test HTML Content %d<p>" % i
+            )
+        self.setup_user(admin=False, enroll=False, login=True)
+        response = self.get_response(usage_id=unicode(self.vetical_block.location))
+        self.assertNotContains(response, '<div class="bookmark-button-wrapper"')
```

Comment:
Thanks for adding the test.  

I'd like the test to be a bit more resilient to changes in the implementation of bookmarks.  One possibility is to use a variable like `bookmark_html = '<div class="bookmark-button-wrapper"'` in include a case where bookmarks do appear (i.e. `self.assertContains(response, bookmark_html)`).  Thus, if implementation changed, the test would fail.  I know that LTI might be an awkward place for this opposite condition though.

Another possibility which is far less fool proof, but makes it slightly more resilient, would be to just search for "bookmark".

Thoughts?